### PR TITLE
Fix editor echo not displaying correctly for lines with whitespace

### PR DESCRIPTION
**Pull request by chiizujin**
#### Brief overview of PR changes/additions

When in normal editing mode (i.e. not in code editing mode) and adding lines with extra whitespace (leading or between words) the input echoing does not correctly reflect what is actually added to the buffer (the extra space is stripped).

This fixes the echo to correctly show what is added.

#### Motivation for adding to Evennia

Correct editor echo display.

#### Other info

**Before (in the editor):**
```
>  This  line   has    whitespace.
01| This line has whitespace.
>:
----------Line Editor [topic test]--------------------------------------------
01|   This  line   has    whitespace.
----------[l:01 w:004 c:0033]------------(:h for help)------------------------
```
**After (in the editor):**
```
>  This  line   has    whitespace.
01|   This  line   has    whitespace.
>:
----------Line Editor [topic test]--------------------------------------------
01|   This  line   has    whitespace.
----------[l:01 w:004 c:0033]------------(:h for help)------------------------
```

### CODE REVIEW

File: evennia/utils/tests/test_eveditor.py [hunk 1]
```diff
@@ -168,13 +168,13 @@ def test_eveditor_COLON_UU(self):
...
             raw_string='First test "line".',
-            msg='01First test "line" .',
+            msg='01First test "line".',
         )
         self.call(
             eveditor.CmdLineInput(),
             "Second 'line'.",
             raw_string="Second 'line'.",
-            msg="02Second 'line' .",
+            msg="02Second 'line'.",
```
Comment (Griatch) [inline evennia/utils/tests/test_eveditor.py:177]: It seems like a good opportunity to actually add a new test here to verify the echo is indeed as expected?

**Review by Griatch — CHANGES_REQUESTED** [review]
Good catch! I agree with the fix, but I think this is also a good opportunity to add a new test to make sure to test this case and make sure to avoid regressions in the future.

**Comment by chiizujin** [comment]
I've added a unit test for this. I added it the end of the editor tests so that the addition of another line doesn't break the tests that rely on the buffer content.

**Review by Griatch — APPROVED** [review]
New test seems fine, thanks for adding it!

**Pull request merged**